docs/BreakingChanges/160 Connection pool blocking period for Asure SQL databases is removed.md

```diff
@@ -0,0 +1,42 @@
+## 160: Connection pool blocking period for Azure SQL databases is removed
```

Comment:
Change file name: "160 Connection pool blocking period for **Asure** SQL databases is removed.md" to "160 Connection pool blocking period for Azure SQL databases is removed.md"